x-pack/plugins/event_log/generated/schemas.ts

```diff
@@ -52,7 +52,7 @@ export const EventSchema = schema.maybe(
         code: ecsString(),
         created: ecsDate(),
         dataset: ecsString(),
-        duration: ecsNumber(),
+        duration: ecsString(),
```

Comment:
Updating this `generated` file directly causes it to be out of sync with the `/x-pack/plugins/event_log/scripts/create_schemas.js` file, which is the documented and recommended way of updating the event log schema. The next time we want to update the event log and we run `node ./x-pack/plugins/event_log/scripts/create_schemas.js`, this duration will be auto-reverted to `ecsNumber()`

Reply:
Good catch, I fixed it in https://github.com/elastic/kibana/pull/130819/commits/0b61084f17c2a3785afb02ff2666ce4c95cd6fb8 by creating `ecsStringOrNumber`. This makes all `long` types in Elasticsearch treated as string | number in Kibana.